keyboards/1upkeyboards/pi40/rules.mk

```diff
@@ -1,26 +1,4 @@
-# MCU name
```

Comment:
This file needs a `DEFAULT_FOLDER` rule or it has to be removed (otherwise it will create invalid build targets).